composer.json

```diff
@@ -78,14 +81,15 @@
     "phpunit:unit": [
       "phpunit --colors --testsuite=unit --do-not-cache-result"
     ],
-    "phpunit:integration": [
-      "phpunit --colors --testsuite=integration --do-not-cache-result"
+    "infection": [
+      "./bin/infection --threads=8 --only-covering-test-cases --min-msi=99 --show-mutations"
```

Comment:
Hard coding to 8 here as various Alpine images / OS setups may not have `nproc` installed and the test suite is fast.

Reply:
But GH worker has only 2 cores :panda_face:

Reply:
Doesn't seem to make it much slower - and much faster locally